js/app.js

```diff
@@ -58,55 +76,54 @@ function btnClickHandler(e) {
   //target --> on we do actually click
 
   if (e.target != e.currentTarget) {
```

Comment:
You can simplify this big if condition.

```js 
if (e.target === e.currentTarget) return; 
// Rest of code
```

Reply:
added this condition to handle the click on currentTarget.